assets/app/scripts/directives/logViewer.js

```diff
@@ -224,6 +228,45 @@ angular.module('openshiftConsole')
               goChromeless: logLinks.chromelessLink,
               restartLogs: streamLogs
             });
+
+            APIDiscovery
+              .getLoggingURL()
+              .then(function(url) {
+                // TODO: update lodash for _.get, would rather do this:
```

Comment:
`$parse` in Angular would work as well if our version of lodash doesn't have `_.get`.